app/controllers/catalog_controller.rb

```diff
@@ -86,7 +86,9 @@ class CatalogController < ApplicationController
 
     # solr field configuration for search results/index views
     config.index.document_presenter_class = IndexDocumentPresenter
-    config.index.title_field = 'title_display'
+
+    # TODO: add a pipeline step that concatenates values
```

Comment:
Is this `TODO` comment still needed?

Reply:
oops, no — that became `TitleRenderingStep`. removed the comment, thanks for catching